docs/src/main/asciidoc/_admin-guide/management-tasks/YAML_Configuration_file.adoc

```diff
@@ -107,3 +107,33 @@ wildfly-configuration:
                 index: 0
 ----
 
+You can't configure everything in YAML, it doesn't currently support:
+
+ - extension: to add extension to the server as this might require modules which can be missing.
+ - deployment: to add *managed* deployments to the server as this requires more than just some configuration.
+ - deployment-overlay: to add deployment-overlays to the server as this requires more than just some configuration.
+ - path: since those should already have been defined when the YAML files are parsed.
+
+While *managed* deployments are not supported you can still define *unmanaged* deployments that are not meant to be managed using the management API which is fine since you are in *read-only* mode with YAML and your changes aren't persisted.
+This is a sample to install two deployments but only one is deployed:
+
+[source,options="nowrap"]
+----
+wildfly-configuration:
+  deployment:
+    test.jar:
+      content:
+        -
+          path: test.jar
+          relative-to: jboss.server.base.dir
+          archive: true
+      enabled: true
+    hello.jar:
+      content:
+        - 
+          path: test.jar
+          relative-to: jboss.server.base.dir
+          archive: true
+----
+
+NOTE: The `content` attribute is of type LIST so you must add a *-* before defining the archive or the directory of your deployment.
```

Comment:
Perhaps

s/The/For historical reasons, the/g

and then add a sentence

'The list should only contain a single element.'